packages/allow-scripts/package.json

```diff
@@ -24,18 +24,22 @@
     "test": "test"
   },
   "devDependencies": {
+    "@metamask/eslint-config-nodejs": "^10.0.0",
     "ava": "^3.15.0",
     "eslint-plugin-ava": "^11.0.0",
-    "eslint-plugin-standard": "^5.0.0"
+    "eslint-plugin-node": "^11.1.0"
   },
   "scripts": {
     "test": "yarn test:run",
     "test:run": "ava --timeout=30s test/index.js",
     "lint": "yarn lint:eslint && yarn lint:deps",
-    "lint:eslint": "eslint \"src/**/*.js\"",
-    "lint:fix": "eslint src/**/*.js --fix",
+    "lint:eslint": "eslint .",
+    "lint:fix": "eslint . --fix",
     "lint:deps": "depcheck"
   },
+  "engines": {
+    "node": ">=12.0.0"
```

Comment:
@kumavis

This was added to address a `node/no-unsupported-features/node-builtins` linting error. However, I think there's more to this. 


It looks like some of the code actually uses some features not in node v12, alread (note: would be good to address those don't get caught in linting, too).

One straight-forward way to resolve this would be to add a constraint to `engines.node: ">=14.x"` is added to `package.json` for each subpackage. So how to treat that comes down to if support for node v12 is actually still desired at this point.

Reply:
Addressed with separate proposal in #398